### DISCUSSION TITLE
Double Xml code Support / replace xml code with custom xml code

### DISCUSSION THREAD

**Discussion by AndroXStudiO** [Sketchware-Pro/Sketchware-Pro#533] [Ideas]
![IMG_20220807_233154](https://user-images.githubusercontent.com/88816748/183303680-4881b5f3-9ee2-485b-8fd7-72ba460facdf.jpg)

**Comment by Arashvscode** [comment]
> ![IMG_20220807_233154](https://user-images.githubusercontent.com/88816748/183303680-4881b5f3-9ee2-485b-8fd7-72ba460facdf.jpg)

Using the command block is not difficult at all

**Comment by AndroXStudiO** [comment]
Make it default why always command block!

**Comment by AndroXStudiO** [comment]
Anyway, i have many imageView in same layout.
And they have different different src... Then how to handle this?

**Comment by AndroXStudiO** [comment]
@JavkhlanK @hasrat-ali 

Sir, this is very much needed,when to want to use dimans files for multi-Screen size App support.
I want to use dimain.xml for Responsive UI. But that very hard to use Command block always for a single XML code (e.g: setTextSize) for every textview

**Reply by j-px** [reply]
If it's so important, you could try writing completely custom XML files. Please don't mention anyone, thanks.

**Comment by j-px** [comment]
You can add e.g. `tools:replace="android:src"` to the inject property for that now, so I guess this feature got implemented.

**Reply by AndroXStudiO** [reply]
okey fine..then how to add multiple double property? 
like -
- android:textColor
- android:textSize
- etc 
at Once... using tools:replace? 
is it support multiple like tools:replace="android:src, android:textSize,android:color......etc"?

**Reply by j-px** [reply]
Yeah, add multiple like this: `tools:replace="android:src, android:textSize"`

**Reply by AndroXStudiO** [reply]
okey then fine❤️
